charts/docker-registry-ui/templates/registry-deployment.yaml

```diff
@@ -51,6 +59,11 @@ spec:
           volumeMounts:
             - mountPath: /var/lib/registry
               name: data
+            {{- if .Values.registry.basicAuth.enabled }}
+            - name: htpasswd
+              mountPath: {{ if .Values.registry.basicAuth.htpasswdPath }}{{ dir .Values.registry.basicAuth.htpasswdPath }}{{ else }}{{ "/auth" }}{{ end }}
```

Comment:
The default value is already in `values.yaml` why should we test it again ?

Reply:
agree with this, except if the value in `values.yaml` are filled with `~`, we should declare default value inside the template.